routes/api/contacts-router.js

```diff
@@ -0,0 +1,87 @@
+import express from "express";
+import {
+  addContact,
+  getContactById,
+  listContacts,
+  removeContact,
+  updateContact,
+} from "../../models/contacts.js";
+import Joi from "joi";
+
+const contactsAddSchema = Joi.object({
+  name: Joi.string().required(),
```

Comment:
Schema must be in separate file. You should validate email and phone properly

Reply:
Thanks, I corrected it. Please check